### PR TITLE
new file:   packages/rankers/rankers.2.0.9/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
version for ocaml-5

### CODE REVIEW

**Comment by mseri** [comment]
This is weird. It is not being installed claiming `molenc` is not available for ocaml 5, yet [molenc 16.15.0](https://github.com/ocaml/opam-repository/blob/master/packages/molenc/molenc.16.15.0/opam) requires ocaml5

File: packages/rankers/rankers.2.0.9/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "ocaml" {>= "5.0.0"}
+  "base-unix"
+  "batteries" {>= "2.9.0"}
+  "bst"
+  "conf-gnuplot"
+  "cpm" {>= "10.0.0"}
+  "dolog" {>= "4.0.0"}
+  "dune" {>= "1.6"}
+  "minicli" {>= "5.0.0"}
+  "molenc"
```
Comment (mseri) [inline packages/rankers/rankers.2.0.9/opam:19]:
```suggestion
  "molenc" {>= "16.15.0"}
```

**Comment by mseri** [comment]
Ok, now the issue is clearer. Molenc needs pyml and this is not available for ocaml 5: `pyml >= 20211015 -> ocaml < 5.0`

**Comment by mseri** [comment]
It should be solved now

**Comment by UnixJunkie** [comment]
Strange, I fixed molenc for ocaml-5 some time ago now.
It was installing fine at the time.

**Comment by UnixJunkie** [comment]
pyml.20220905 installs fine with ocaml-5

**Comment by UnixJunkie** [comment]
on ubuntu and w/ ocaml-5, I confirm rankers.2.0.9 can install

**Pull request merged**